tuna.go

```diff
@@ -832,6 +858,16 @@ func checkPaymentTimeout(session *smux.Session, updateTimeout time.Duration, las
 		}
 		lastCost = totalCost
 
+		totalTraffic = getTotalTraffic()
```

Comment:
The logic is not right here. If there are some small traffic after tuna conn is established, the service provider side will disconnect after 2s because user side has not pay anything yet. Have you really tested it yet?

Reply:
It's better not to change `checkPaymentTimeout`, but create another function:
```
for every second {
    if totalTraffic - lastPaidTraffic > threshold / MinTrafficCoverage {
        // fail to pay logic
    }
}
```